specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/scheduledActions.json

```diff
@@ -0,0 +1,940 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "checkNameAvailabilityRequest",
+            "description": "Scheduled action to be created or updated.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/CheckNameAvailabilityRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response of scheduled action name availability.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/CheckNameAvailabilityResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DaysOfWeek": {
+      "type": "string",
+      "description": "Days of Week.",
+      "enum": [
+        "Monday",
+        "Tuesday",
+        "Wednesday",
+        "Thursday",
+        "Friday",
+        "Saturday",
+        "Sunday"
+      ],
+      "x-ms-enum": {
+        "name": "DaysOfWeek",
+        "modelAsString": true
+      }
+    },
+    "ErrorResponse": {
+      "description": "Error response indicates that the service is not able to process the incoming request. The reason is provided in the error message. \n\nSome Error responses: \n\n * 429 TooManyRequests - Request is throttled. Retry after waiting for the time specified in the \"x-ms-ratelimit-microsoft.consumption-retry-after\" header. \n\n * 503 ServiceUnavailable - Service is temporarily unavailable. Retry after waiting for the time specified in the \"Retry-After\" header.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "The details of the error.",
+          "$ref": "#/definitions/ErrorDetails"
+        }
+      }
+    },
+    "ErrorDetails": {
+      "description": "The details of the error.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "FileDestination": {
+      "type": "object",
+      "description": "Destination of the view data. This is optional. Currently only csv format is supported.",
+      "properties": {
+        "fileFormats": {
+          "type": "array",
+          "description": "Destination of the view data. Currently only csv format is supported.",
```

Comment:
>csv [](http://example.com/codeflow?start=71&length=3)

For docs, should this be '.csv' 'CSV' or 'comma separated variables (.csv)'? What do other microsoft doc pages do?

Reply:
Exports API uses Csv - [https://learn.microsoft.com/en-us/rest/api/cost-management/exports/create-or-update?tabs=HTTP](https://learn.microsoft.com/en-us/rest/api/cost-management/exports/create-or-update?tabs=HTTP)

Stream analytics uses Csv as well - [https://learn.microsoft.com/en-us/rest/api/streamanalytics/2020-03-01/outputs/create-or-replace?tabs=HTTP](https://learn.microsoft.com/en-us/rest/api/streamanalytics/2020-03-01/outputs/create-or-replace?tabs=HTTP)

Reply:
Will update it to "Csv".